src/main/java/duke/command/GetExternalListCommand.java

```diff
@@ -0,0 +1,37 @@
+package duke.command;
+
+import duke.dukeobject.ExpenseList;
+import duke.exception.DukeException;
+import duke.parser.CommandParams;
+import duke.ui.Ui;
+
+public class GetExternalListCommand extends Command {
```

Comment:
should be 3 separate classes so that help can be implemented later; the implementation of all commands should resemble the current add and delete commands.